nb2workflow/nbadapter.py

```diff
@@ -683,22 +686,26 @@ def _execute(self, parameters, progress_bar=True, log_output=True, inplace=False
                        log_output = True,
                        cwd = tmpdir,
                     )
-                except pm.PapermillExecutionError as e:
+                except (pm.PapermillExecutionError, DeadKernelError) as e:
                     exceptions.append([e,e.args])
                     logger.info(e)
                     logger.info(e.args)
 
-                    if e.ename == "WorkflowIncomplete":
+                    if getattr(e, 'ename', None) == "WorkflowIncomplete":
                         logger.info("detected incomplete workflow")
                         self.update_summary(state="incomplete dependency", dependency=repr(e))
                         raise  PapermillWorkflowIncomplete()
```

Comment:
and in the case `e` does not contain `ename` ? or the it is not `WorkflowIncomplete`?

should we also send a sentry ?

Reply:
I don't think we want sentry for any "normal"/"expected" exception in a notebook. 

For `DeadKernelError` it probably makes sense to send it as it is a bit special condition

This treatment for `WorkflowIncomplete` was there before, I don't really know what's this about, and when does it happen. Maybe @volodymyrss knows.